core/common/validator-core/src/main/java/org/eclipse/edc/validator/jsonobject/validators/LogDeprecatedValue.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (c) 2024 T-Systems International GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       T-Systems International GmbH - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.validator.jsonobject.validators;
+
+import jakarta.json.JsonObject;
+import org.eclipse.edc.spi.monitor.Monitor;
+import org.eclipse.edc.validator.jsonobject.JsonLdPath;
+import org.eclipse.edc.validator.spi.ValidationResult;
+import org.eclipse.edc.validator.spi.Validator;
+
+import java.util.Optional;
+
+import static java.lang.String.format;
+
+/**
+ * Verify that a @value is present but deprecated.
+ */
+public class LogDeprecatedValue implements Validator<JsonObject> {
+    public final String deprecatedLog;
+    private final JsonLdPath path;
+    private final Monitor monitor;
+
+    public LogDeprecatedValue(JsonLdPath path, String deprecatedLog, Monitor monitor) {
```

Comment:
this constructor is called only once: by the other constructor, please inline

Reply:
I've inlined and removed the unreferenced constructor.